webapp/src/pages/app/projects/[projectName]/index.tsx

```diff
@@ -16,6 +19,18 @@ const ProjectDetailPage = () => {
     refetchOnMountOrArgChange: true,
     skip: false,
   });
```

Comment:
It would be worth fetching the data for the active tab.
What do you think of adding some if-else conditions or a map that associates tabs and data API slices?

Reply:
Updated to `refetchOnFocus`. 

The mapping between the slices and tabs are not 1:1. `metadata` is used by `permissions`, `members` and `tokens' tabs.